src/createServer.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const cors = require('cors');
+const cookieParser = require('cookie-parser');
+
+const { errorMiddleware } = require('./middlewares/errorMiddleware');
+const { authRouter } = require('./routes/auth.route');
+const { usersRouter } = require('./routes/users.route');
+
+function createServer() {
+  const app = express();
+
+  /* CORS  */
+  const options = {
+    origin: true,
+    methods: 'GET, POST, PUT, DELETE , PATCH',
+    allowedHeaders: 'Content-Type',
```

Comment:
Consider including additional headers in `allowedHeaders` if your application requires them, such as 'Authorization' or other custom headers.